dts/bindings/auxdisplay/maxim,max7219-7seg.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2025 Maksim Salau <maksim.salau@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+title: Maxim MAX7219 8-Digit LED Display Driver
+
+description: |
```

Comment:
would stick with original point that there should be 1 dts file with 1 compatible and auxdisplay/LED can both use it, can be put in `mfd` folder, and needs both these properties and the ones from the other file, then the Kconfigs, etc. can be updated